portfolio/src/main/webapp/script.js

```diff
@@ -38,8 +39,35 @@ function FloatingNavbar() {
   }
 }
 
-function getBookRec() {
-  fetch('/literature').then(response => response.text()).then((quote) => {
-    document.getElementById('book-container').innerText = quote;
-  })
+/* Fetches comments from DataServlet and display */
+function getComments() {
+  fetch('/comment').then(response => response.json()).then((commenthistory) => {
+    const historyEl = document.getElementById('history');
```

Comment:
Minor, but in general prefer to not using abbreviations which are not immediately recognizable to the reader. historyElement or at least historyElem would be more readable. Especially as l is easy to parse as i or 1 depending on the font.

Reply:
Ok, I'll fix that in the future